### PR TITLE
Finished about page (I forgot to create a new branch for this)

### PR DESCRIPTION
**Pull request by yaaowusu00**
Hi guys, let me know what you think of my design.  There is a grid with all 4 of our pictures **(Oscars picture is only there twice because it s a placeholder for Gi's picture)**. When you click on a picture, the persons bio will show up in the middle of the grid (will insert gif). I started working on a mobile design (will insert gif) and I like how the pictures look, but when the user clicks I don't think the bio information should be displayed in between the pictures on mobile because its not super intuitive on a phone screen. Next week I'll probably think of a better way to change the css for mobile to display the bios.

### CODE REVIEW

**Comment by yaaowusu00** [comment]
I tried converting my videos to gifs but it said they were too big so I'll just link the videos for now.
mobile: https://drive.google.com/file/d/1i-gPwTxLZ1WC0qrWGxSlXxKPA78ufX8q/view?usp=sharing
regular: https://drive.google.com/file/d/1XkbqRJ7wyiSo0meethfd3X3i8qNdCXhd/view?usp=sharing

File: fudge-ratio/src/main/webapp/about/about.css [hunk 1]
```diff
@@ -2,52 +2,203 @@
...
+  --base: min(3.5vw, 1.7rem);
 }
-
 body { 
-  margin:0; 
+  margin:0;
   background: var(--background-color);
 }
-
+/* Style for the bar on top and text on top*/
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/about.css:14]: Thanks for the comments n.n

File: fudge-ratio/src/main/webapp/about/about.css [hunk 2]
```diff
@@ -2,52 +2,203 @@
...
 .banner {
-  overflow: hidden;
-  /* 161, 111, 32, 0.712*/
+  z-index: 3;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/about.css:16]: One quick question, why we need z-index = 3? 3 seems like a weird magical constant

File: fudge-ratio/src/main/webapp/about/about.css [hunk 3]
```diff
@@ -2,52 +2,203 @@
...
+  width: 100%;
+  height: 12vh;
+  padding-top:2%;  
+  padding-left:2%; 
 }
 
-#logo{
+#logo {
+  z-index: 4;
+  font-family: 'Poiret One', sans-serif;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/about.css:31]: Another quick thing I am not getting, where are you importing the ``'Poiret One'`` font family?

File: fudge-ratio/src/main/webapp/about/about.css [hunk 4]
```diff
@@ -2,52 +2,203 @@
...
+  color: rgb(229, 231, 225);
+}
+#Gi {
+  background-image: url("oscar_pic.jpg");
+}
+#Jerry {
+  background-image: url("jerry_pic.jpg");
+}
+#Oscar {
+  background-image: url("oscar_pic.jpg");
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/about.css:98]: Really cool way of importing assets

File: fudge-ratio/src/main/webapp/about/index.html [hunk 5]
```diff
@@ -9,25 +9,73 @@
...
+                <div id="name">Gi</div>
+            </div>
+            <div class="pictureBox" id="Jerry" onclick="openTab('jerryBio');">
+                <div id="triangle"></div>
+                <div id="name">Jerry</div>
+            </div>
+        </div>
+
+        <!-- The expanding grid (hidden by default) -->
+        <div id="giBio" class="bioContainer" >
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/index.html:30]: I really like that all the information is already here, the html makes so much sense

File: fudge-ratio/src/main/webapp/about/index.html [hunk 6]
```diff
@@ -9,25 +9,73 @@
...
+            <span onclick="this.parentElement.style.display='none'" class="closebtn">x</span>
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/index.html:31]: this onclick seems like something that should only be handled in a js file, but i am open to opinions

File: fudge-ratio/src/main/webapp/about/about.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+function openTab(tabName) {
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/about.js:1]: This could just be written like:
```
const openTab = tabName => 
  Array.from(document.getElementsByClassName("bioContainer"))
    .forEach(bio => bio.style.display = bio.id === tabName? "block": "none")
```

In this way we can see more clearly the flow of the data, how we are moving one by one on elements of an array, how we are changing the style of the elements based on the id

File: fudge-ratio/src/main/webapp/landing.css [hunk 8]
```diff
@@ -11,23 +11,32 @@ body {
   background: var(--background-color);
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/landing.css:11]: Why is this changes here, if the commits talks only about the "About page" and there is no mention of modifying the landing page?
Can you show us the before and after of the landing page? 😊🙏🏻

File: fudge-ratio/src/main/webapp/about/about.css [hunk 9]
```diff
@@ -2,52 +2,203 @@
...
   background: var(--background-color);
 }
-
+/* Style for the bar on top and text on top*/
 .banner {
-  overflow: hidden;
-  /* 161, 111, 32, 0.712*/
+  z-index: 3;
+  word-wrap: break-word;
+  backdrop-filter: blur(2rem);
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/about.css:18]: OMG! I am in love with the blur, so cool 😎

File: fudge-ratio/src/main/webapp/about/about.css [hunk 10]
```diff
@@ -2,52 +2,203 @@
...
   color: rgb(37, 51, 51);
-  width: 90vw;
-  height: 20vh; 
 }
+/*ending of style for top part of page*/
 
-.bio {
+/*Beginning of style for the expandable grid*/
+ .row {
+  border-radius: 5vw;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/about.css:66]: Do we need border-radius here?

File: fudge-ratio/src/main/webapp/about/about.css [hunk 11]
```diff
@@ -2,52 +2,203 @@
...
+  position: relative;
+  width: 64vw;
+  display: flex;
+  flex-direction: row;
+  overflow: hidden;
+  justify-content: center;
+  background: none;
+  margin: 0 auto;
+ }
+.pictureBox {
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/about/about.css:76]: I was so interested on why there are so many properties, after playing with the code for a while here are my conclusions:

The logic in here is so brilliant, the way you created the grid or the triangle, so nice,
The thing is that CSS now have properties for things like these, so we do not longer need hacks:
Here is the result of me playing for a while:
```
/*Beginning of style for the expandable grid*/
 .row {
  width: 64vw;
  margin: 1rem auto;
  display: grid;
  grid-template-columns: 1fr 1fr;
  grid-auto-flow: column;  
  height: 40vw;
  grid-gap: 1rem;
 }

.pictureBox {
  border-radius: 2vw;
  background-size: cover; 
  cursor: pointer;
}

#triangle { }

#name{  
  color: rgb(229, 231, 225);
  z-index: 2; 
  font-weight: bold;
  font-family: 'Poiret One', sans-serif;
  font-size: 6vh;
  vertical-align: top;
  border-top-left-radius: 2vw;
  width: 26vh;
  height: 14vh;
  padding: 0.7rem;
  background: linear-gradient(to right bottom, #3553129a 50%, #ffffff00 50%);
}
```

## Note that we are using css grid (really important topic) and linear-gradient.
## Delete a lot of properties that was set to their default

Here are some pics:
Screen Shot 2020-04-12 at 2 57 48 PM


Screen Shot 2020-04-12 at 2 58 04 PM

They give more or less the same result with a lot less but clearer code

Comment (SoyOscarRH) [reply fudge-ratio/src/main/webapp/about/about.css:76]: This is not perfect, I just played with the code for 20 minutes, but if you are interested in this ping me

File: fudge-ratio/src/main/webapp/about/about.css (hunk 4 above)
Comment (SoyOscarRH) [reply fudge-ratio/src/main/webapp/about/about.css:98]: https://codepen.io/forthewinn/pen/OVapJw

File: fudge-ratio/src/main/webapp/about/about.css (hunk 2 above)
Comment (yaaowusu00) [reply fudge-ratio/src/main/webapp/about/about.css:16]: because when the banner scrolls over everyones pictures, it haze to say on top, so it needs a higher z index than the pictures and the triangles for it to stay on top. The text in the banner has the highest z index so that that is always on top of the banner and everything else in the page too

**Review by SoyOscarRH — APPROVED** [review]

**Review by gilemos — APPROVED** [review]
Great work Yaa! I loved how clever the design is with the banners!
